packages/nodejs/package.json

```diff
@@ -26,7 +26,7 @@
     "typescript": "~> 4.1"
   },
   "optionalDependencies": {
-    "@appsignal/nodejs-ext": "~> 1.2"
+    "@appsignal/nodejs-ext": "=1.2.3-beta.0"
```

Comment:
I think it's best to specify the exact version we want here from now on. If there is a mismatch, for example the package is in beta, the linking files and it falls back to the version from the registry.